stdlib/Random/src/normal.jl

```diff
@@ -58,7 +61,7 @@ end
 
 @inline function _randn(rng::AbstractRNG, r::UInt64)
     @inbounds begin
-        r &= 0x000fffffffffffff
+        r = _take52(rng, r)
```

Comment:
Here using `_take52` is fine, although this `_randn` function is mostly useful for `randn!`, so you could as well leave the code untouched here, and instead right-shift by 12 positions in `rand!` itself.